airlock/file_browser_api.py

```diff
@@ -561,12 +562,29 @@ def build_path_tree(
                 # ReleaseRequest containers, so we have these container specfic
                 # calls
                 node.workspace_status = container.get_workspace_status(path)
-                node.request_status = container.get_request_status(path)
 
-                if user:
-                    node.user_request_status = container.get_user_request_status(
-                        path, user
-                    )
+                if isinstance(container, ReleaseRequest):
+                    if container.status in [
+                        RequestStatus.SUBMITTED,
+                        RequestStatus.PARTIALLY_REVIEWED,
+                    ]:
+                        # if the request is in independent review phase, set the user review
+                        # status for the file
+                        if user:  # pragma: no cover
```

Comment:
Right, this is basically the same logic I have, but in a different place.

I can handle this when I rebase on this once its landed